Update data node management with example of moving a chunk (#174)
timescale/docs: timescaledb/how-to-guides/distributed-hypertables/manage-data-nodes.md

 conditions      | {data_node_1,data_node_2,data_node_3}
```

> If additional data nodes are added to a database, the data nodes are
> not automatically associated with existing distributed
> hypertables. Instead, you need to explicitly *attach* a data node
> using [`attach_data_node`][attach_data_node]:

Reviewer: ```suggestion
If you add additional data nodes to a database, they are
not automatically associated with existing distributed
hypertables. Instead, you need to explicitly attach a data node
using [`attach_data_node`][attach_data_node]:
```

> When attaching a data node, the partitioning configuration of the

Reviewer: ```suggestion
When you attach a data node, the partitioning configuration of the
```

distributed hypertable is also updated to account for the additional

> data node (i.e., the number of space partitions is automatically
> increased to match), unless the function parameter `repartition` is
> set to `FALSE`. The updated partitioning configuration ensures that
> the data node will be able to take on newly created chunks.

Reviewer: ```suggestion
data node (the number of space partitions are automatically
increased to match), unless the function parameter `repartition` is
set to `FALSE`. The updated partitioning configuration ensures that
the data node is able to take on newly created chunks.
```

all its data is deleted first, or (2) the data is replicated on other
data nodes (see the next section on native replication).

## Moving data between nodes

> When attaching a new data node to a distributed hypertable, it might
> desirable to move data to the new node in order to free up storage on
> existing nodes and make queries use the added capacity. Data can be
> moved between nodes at the chunk level:

Reviewer: ```suggestion
When you attach a new data node to a distributed hypertable, you might
prefer to move data to the new node to free up storage on the
existing nodes and make queries use the added capacity. Data can be
moved between nodes at the chunk level:
```

```sql
CALL timescaledb_experimental.move_chunk('_timescaledb_internal._dist_hyper_1_1_chunk', 'data_node_3', 'data_node_2');

```

> <highlight type="warning"> 
> The ability to move chunks between data nodes is an experimental
> feature that is under active development. Use in production
> environments at your own risk.
> </highlight>

Reviewer: ```suggestion
<highlight type="warning"> 
The ability to move chunks between data nodes is an experimental
feature that is under active development. We recommend that you
do not use this feature in a production environment.
</highlight>
```
